pyaedt/application/Analysis3DLayout.py

```diff
@@ -53,6 +53,8 @@ class FieldAnalysis3DLayout(Analysis):
     aedt_process_id : int, optional
         Only used when ``new_desktop_session = False``, specifies by process ID which instance
         of Electronics Desktop to point PyAEDT at.
+    ic_mode : bool, optional
+        Whether to set the design to IC mode or not. The default is ``False``.
```

Comment:
```suggestion
        Whether to set the design to IC mode. The default is ``False``.
```
Is the part about applying only HFSS 3D Layout not needed here. It seems like the descriptions should be consistent.